test/collection_test.js

```diff
@@ -62,6 +62,21 @@ describe("Collection", () => {
     };
   }
 
+  function createKeyListIdSchema() {
+    return {
+      generate(record) {
+        let id = [];
+        for (let key in record) {
+          id.push(key);
+        }
+        return id;
```

Comment:
Looking at this again I remembered why I didn't use the example from the docs. When I tried to run the tests from the CLI they failed because Node doesn't have the `btoa` and `atob` function available. When I tried using the recommendation from [here](http://stackoverflow.com/a/23097961) it threw a different error, so I went with the test that you see here. I'll take another crack at building a more useful example that isn't quite so strange.